source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -521,6 +542,11 @@ Http::FilterHeadersStatus Filter::encodeHeaders(Http::ResponseHeaderMap& headers
       }
     }
   }
+
+  if (omitted_response_headers) {
+    stats_.omitted_response_headers_.inc();
```

Comment:
Unless, you want to change the logs to include which header was dropped, I think the log should be moved into this block so it only logs once per stream, not once per header.